modules/conflict_resolver/php/conflict_resolver.class.inc

```diff
@@ -285,7 +285,7 @@ class Conflict_Resolver extends \NDB_Menu_Filter_Form
 
         // Get instruments
         $instruments = \Utility::getAllInstruments();
-        $instruments = array_merge(array('' => 'All Instruments'), $instruments);
+//        $instruments = array_merge(array('' => 'All Instruments'), $instruments);
```

Comment:
remove this whole line instead of commenting out?

Reply:
Project should also have either empty option or all option, not both

Reply:
@zaliqarosli thanks! I added them.